Dockerfiles/mariadb/db-encrypt/keys.txt

```diff
@@ -0,0 +1,100 @@
+1;58b22a41c9d095b8003995cf726f852a2c2bd44e4be5da2f52f643d7559c574d
```

Comment:
I think this file is not needed as you have encrypted this into keys.enc.

Reply:
I agree, but decided to keep it to make it clear what's going on. These files are mostly intended as a reference to show what is needed in production, and thus I thought it made sense to include the unencrypted file to remind me about the needed formatting etc.

Reply:
Also: technically only one key is needed, but I chose to follow their instructions by the letter when I set it up. It seems it chooses to use a different key for e.g. tmp files, so I guess five or so is probably a good number. The plugin does not support key rotation, so no need to keep them for that.

Reply:
Yes I too saw that key rotation is not available and the key ids don't need to be contiguous. Perhaps you can reduce the number of keys?
I think documentation is a separate work, which can be kept either next to the actual thing or at a separate place. Since these are keys, I think it is better to keep it with its(your, not the online one) documentation elsewhere, so that you don't run into the issue of remembering whether it was needed or not to be in this place. This is a suggestion. You run this, so final choice is yours.